recipes/newrelic/infrastructure/kubernetes.yml

```diff
@@ -411,11 +411,13 @@ install:
                 fi
                 if [[ "$UNINSTALL_ANSWER" == "N" ]]; then
                   echo "Attempting install over existing integration."
+                  echo "{\"Metadata\":{\"uninstallNriBundleAnswer\":\"No\"}}" | tee -a {{.NR_CLI_OUTPUT}} > /dev/null
```

Comment:
can you add metadata for "PreviousNRIBundleDetected" : "yes" before the while loop in this block, and then edit this block to have an `else` statement at the end that adds  "PreviousNRIBundleDetected" : "no"?

Reply:
Done